app/view/todo.py

```diff
@@ -104,6 +103,12 @@ def todo():
         'bumped_groups': bumped_groups,
         'orphan_issues': orphan_issues
     }
+
+
+@app.route('/todo', methods=['GET'])
```

Comment:
can we get a json endpoint like all other endpoints have?

Reply:
Sure, do we want to split them based on the 'todo type', so scheduled, incomplete advisories?